Library/Formula/commonmark.rb

```diff
@@ -0,0 +1,51 @@
+class Commonmark < Formula
+  url "https://github.com/jgm/CommonMark/archive/0.15.tar.gz"
+  homepage "http://commonmark.org"
+  sha1 "877665a96fdc5fcc42ec2cd605d8535344a27b72"
+
+  depends_on "cmake" => :build
+
+  def install
+    # a build directory is necessary since building from the source
+    # directory is forbidden.
+    mkdir "build"
+    Dir.chdir("build") do
+      system "cmake", "..", "-DCMAKE_INSTALL_PREFIX=#{prefix}"
+
+      # For some reason, running
+      #
+      #     system "make"
+      #
+      # could result in wrong build order (man before libcmark) and
+      # cause an error, so I'm manually making each target here.
+      system "make", "cmark", "libcmark", "api_test", "man"
+
+      # run the test suite
+      system "make", "test"
+
+      if $?.exitstatus == 0
```

Comment:
`Formula#system` will raise if exitstatus is non-zero so you don't need this conditional.